CHANGES.md

```diff
@@ -7,6 +7,7 @@
 - Improve error message for invalid regular expression (#2678)
 - Fix mapping cases that contain as-expressions, like `case {"key": 1 | 2 as password}`
   (#2686)
+- `--verbose` enhancements (project root, how is config found) (#2526)
```

Comment:
Perhaps instead:

> Verbose mode also now describes how a project root was discovered and which paths will be formatted. (#2526) 

This feels a bit more concrete than what we have right now but then again this might be too technical?